src/app/shared/components/template/services/template-metadata.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable, OnDestroy, signal } from "@angular/core";
+import { SyncServiceBase } from "src/app/shared/services/syncService.base";
+import { TemplateService } from "./template.service";
+import { FlowTypes } from "src/app/shared/model";
+import { ActivatedRoute, NavigationEnd, Router } from "@angular/router";
+import { distinctUntilChanged, filter, map, Subject, takeUntil } from "rxjs";
+import { Capacitor } from "@capacitor/core";
+
+/**
+ * Service responsible for handling metadata of the current top-level template,
+ * i.e. parameters authored through the template's parameter_list in a contents list
+ */
+@Injectable({
+  providedIn: "root",
+})
+export class TemplateMetadataService extends SyncServiceBase implements OnDestroy {
+  public parameterList = signal<FlowTypes.Template["parameter_list"]>({});
+  private enabled: boolean;
+  private destroy$ = new Subject();
+
+  constructor(
+    private templateService: TemplateService,
+    private router: Router
+  ) {
+    super("TemplateMetadata");
+
+    // Currently the only watched parameter is for screen orientation,
+    // which is only supported on native platforms
+    this.enabled = !Capacitor.isNativePlatform();
```

Comment:
nit(blocking): I don't think it makes sense to define the service enable/disable state relative to a child dependency (really this service should have no knowledge/awareness of others consuming it)